modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -12,6 +12,20 @@ You can create a virtual machine (VM) snapshot for an offline or online VM by cr
 
 * Ensure that the persistent volume claims (PVCs) are in a storage class that supports Container Storage Interface (CSI) volume snapshots.
 * Install the OpenShift CLI (`oc`).
+* Ensure the `Snapshot` feature gate is enabled for the `HyperConverged` CR.
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get hyperconverged kubevirt-hyperconverged -n {CNVNamespace} -o yaml
+----
++
+.Truncated output
+[source,yaml]
+----
+spec:
+  featureGates:
+    - Snapshot
+----
 * Optional: Power down the VM for which you want to create a snapshot.
 
 .Procedure
```

Comment:
In line 57:
> Optional: If you are taking an online snapshot, you can use the `wait` command and monitor the status of the snapshot:

You dont need an online snapshot to use the wait command, and also the point is not exactly to monitor the status but rather wait for the snapshot to be ready to use.

Reply:
in line 65 and down starting with
> Verify the status of the snapshot:

This is more part of the `Verification` so I think it should be moved there.
The example output is not very recent and there are new fields in the snapshot that can be used for the verification. 
In this new fields there are the `indications` and `included volumes` which the user should look and verify that everything he expected to have in the snapshot is there.

Reply:
in line 132
> . Check the `spec:volumeBackups` property of the `VirtualMachineSnapshotContent` resource to verify that the expected PVCs are included in the snapshot.
This can be done as mentioned with the `included volumes` in the snapshot status so no need to go check with the `VirtualMachineSnapshotContent`